android/guava-tests/test/com/google/common/cache/CacheBuilderTest.java

```diff
@@ -599,7 +600,9 @@ public String load(String key) throws InterruptedException {
             .removalListener(removalListener)
             .maximumSize(5000)
             .build(countingIdentityLoader);
-
+    
+    // Refactoring this causes test to hang for a while and eventually give wrong result
+    // https://github.com/ponder-lab/guava/actions/runs/9736322153/job/26866789783#step:6:328
```

Comment:
Refactoring this causes [test to hang for a while and eventually give wrong result](https://github.com/ponder-lab/guava/actions/runs/9736322153/job/26866789783#step:6:328)